engine/src/main/java/io/zeebe/engine/state/instance/ElementInstance.java

```diff
@@ -15,10 +15,13 @@
 import io.zeebe.msgpack.property.ObjectProperty;
 import io.zeebe.protocol.impl.record.value.workflowinstance.WorkflowInstanceRecord;
 import io.zeebe.protocol.record.intent.WorkflowInstanceIntent;
+import io.zeebe.util.sched.clock.ActorClock;
 
 public final class ElementInstance extends UnpackedObject implements DbValue {
 
   private final LongProperty parentKeyProp = new LongProperty("parentKey", -1L);
+  private final LongProperty creationTimeProp =
+      new LongProperty("creationTime", ActorClock.currentTimeMillis());
```

Comment:
Hm, don't know how I feel about using the ActorClock as a default, which has different behaviour based on which thread calls it :sweat_smile: 

Also, what about existing instances? Will this metric give me initially incredibly fast jobs since creation time will be instantiated with the current timestamp? :sweat_smile:

Reply:
> Hm, don't know how I feel about using the ActorClock as a default, which has different behaviour based on which thread calls it sweat_smile

What is the alternative? You want to use system time? I mean we did in our test but actually the clock was introduced to avoid calling system time to often.


> Also, what about existing instances? Will this metric give me initially incredibly fast jobs since creation time will be instantiated with the current timestamp?

dont get this question to be honest